### ISSUE TITLE
#22 on Midterm 3 Practice

### ISSUE THREAD

**Issue by hfmandell** [mikeizbicki/cmc-csci145-math166#134]
Question 22: 

> In scikit-learn's `sklearn.tree.DecisionTreeClassifier` model, increasing the value of the hyperparamater `max_depth` increases the VC dimension.

The answer in`midterm-practice.tex` is False. However, I thought that increasing `max_depth` would indeed increase the VC dimension because VC dim = O(2^j jlogd), where j is the height of a decision tree.

**Comment by mikeizbicki** [comment]
You are correct, and this is a mistake in the tex file.
